NanoAnalysis/test/runLocal.py

```diff
@@ -200,6 +188,21 @@ def customizeProcessForLocal(p) :
     
 setConf("customizations", customizeProcessForLocal, append=True) 
 
+=======
```

Comment:
This entire block (L191-L205) is a leftover from a git conflict and should be removed